File: CS/DXApplication1/WebEUD/Helpers/DataHelpers.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace System.Data {
    static class DataHelpers {
        public static void AddColumns(this DataTable source, params DataColumn[] columns) {
            source.Columns.AddRange(columns);
        }

        public static void AddRow(this DataTable source, params object[] values) {
            source.Rows.Add(values);
        }
    }

}

File: VB/DXApplication1/WebEUD/Helpers/DataHelpers.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Namespace System.Data
    Friend Module DataHelpers
        <System.Runtime.CompilerServices.Extension> _
        Public Sub AddColumns(ByVal source As DataTable, ParamArray ByVal columns() As DataColumn)
            source.Columns.AddRange(columns)
        End Sub

        <System.Runtime.CompilerServices.Extension> _
        Public Sub AddRow(ByVal source As DataTable, ParamArray ByVal values() As Object)
            source.Rows.Add(values)
        End Sub
    End Module

End Namespace